app/controllers/api/teams.php

```diff
@@ -343,11 +343,12 @@
             Query::limit(2000), // TODO fix members limit
         ]);
 
-        // TODO delete all members individually from the user object
         foreach ($memberships as $membership) {
+            // Memberships are deleted here instead of in the worker to make sure user permisions are updated instantly
```

Comment:
Would you please move the comment to before the for loop because the comment applies to the whole block?